src/index.html

```diff
@@ -2,17 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Product cards</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product card</title>
+    <link rel="stylesheet" href="./styles/index.scss" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
-      rel="stylesheet"
-      href="./styles/index.scss"
-    />
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet" />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="product-card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock" class="product-card__img" />
```

Comment:
`Lines of code have 80 chars max`
```suggestion
      <img 
        src="./images/imac.jpeg" 
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock" 
        class="product-card__img"
       />
```